debian/control

```diff
@@ -203,6 +203,7 @@ Replaces: ceph-client-tools,
 	  librbd1 (<< 0.92-1238)
 Breaks: ceph (<< 9.0.0-943),
 	python-ceph (<< 0.92-1223),
+	ceph-test (<< 9.0.3-1483),
```

Comment:
Today I've run more tests with apt-get, upgrading from Hammer to this version, and I can find no breakages so far. We are good here without adding any explicit `Breaks:` / `Replaces:` to ceph-common-dbg.